examples/components/Radar.vue

```diff
@@ -29,7 +42,10 @@ export default {
     },
     handleDirectionChange(angle) {
       // todo need better solution
-      const shape = this.$parent.getStage().children[0]
+      // const shape = this.$parent.getStage().children[0]
+
+      // got better solution now
+      const shape = this.$refs.wedge.getStage()
 
       var tween = new Konva.Tween({
```

Comment:
Here you should make `tween` an instance variable and stop it every time before recreating a new one.

Also use `let` instead of `var` to avoid variable scope collisions.